### DISCUSSION TITLE
Add `better-sqlite` support

### DISCUSSION THREAD

**Discussion by daawaan4x** [directus/directus#16468] [Feature Requests]
I am currently using directus on top of sqlite for a very lightweight deployment that has to run in a 1 CPU Core 512MB RAM VPS. I've had random momentary freezes and connection pooling errors popping up every now and then and since `knex` also currently supports `better-sqlite`, I was wondering if it would be fine to also support `better-sqlite`

**Comment by rijkvanzanten** [comment]
> had random momentary freezes and connection pooling errors popping up every now and then

Is this confirmed to be caused by node-sqlite3, or could it be something else? Just wanna make sure before we do any R&D on better-sqlite 🙂

**Reply by daawaan4x** [reply]
For unknown reasons, setting pragma `journal_mode = wal` and `synchronous = normal` fixed said issues

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
